pkgs/development/libraries/dav1d/default.nix

```
@@ -10,14 +10,13 @@ assert useVulkan -> withExamples;
 
 stdenv.mkDerivation rec {
   pname = "dav1d";
-  version = "1.0.0";
+  version = "1.1.0";
 
-  src = fetchFromGitLab {
-    domain = "code.videolan.org";
+  src = fetchFromGitHub {
     owner = "videolan";
     repo = pname;
-    rev = version;
-    sha256 = "sha256-RVr7NFVxY+6MBD8NV7eMW8TEWuCgcfqpula1o1VZe0o=";
+    rev = "${version}";
```

```suggestion
    rev = version;
```
why not leave it like that?

---

This is now fixed in commit 4040da1900d3c7c6e3a273a2b6fa94fdeb2dfe55. 
I also changed the "sha256" to "hash", as you suggested in another PR here: https://github.com/NixOS/nixpkgs/pull/217986#discussion_r1120633319